src/app/genes/gene-similar/gene-similar.component.ts

```diff
@@ -316,13 +312,21 @@ export class GeneSimilarComponent implements OnInit {
         return druggability[0].pharos_class;
     }
 
-    druggability(druggability: Druggability[]): string {
-        if (!druggability || !druggability.length || !druggability[0].sm_druggability_bucket ||
-            !druggability[0].classification
-        ) {
+    druggability(druggability: Druggability[], subfield: string): string {
+        if (!druggability || !druggability.length || !druggability[0][subfield]) {
             return '-';
+        } else {
+            switch (subfield) {
```

Comment:
consider a `default` branch that shows a sensible error/placeholder, depending on how confident you are in the above conditional and the type/value safety of this string

Reply:
Done!